lib/LMS.class.php

```diff
@@ -2474,6 +2474,16 @@ public function EventOverlaps(array $params) {
 		return $manager->EventOverlaps($params);
 	}
 
+    public function AssignCurrentUserToEvent($id, $userid) {
+        $manager = $this->getEventManager();
+        return $manager->AssignCurrentUserToEvent($id, $userid);
+    }
+
+    public function DeassignCurrentUserFromEvent($id, $userid) {
```

Comment:
Nie ma czegoś takiego jak Deassign - jeśli już Unassign.

Reply:
Poza tym dlaczego w nazwie metody dajesz CurrentUser skoro $userid jest jako parametr?
Jeśli już to:
AssignUserToEvent()
UnassignUserFromEevent();